packages/xrpl/src/Wallet/fundWallet.ts

```diff
@@ -18,10 +19,28 @@ interface FaucetWallet {
   balance: number
 }
 
+interface HooksV2FaucetWallet {
+  address: string
+  secret: string
+  xrp: number
+  hash: string
+  code: string
+}
```

Comment:
Would it be possible to have comments explaining what each of these fields is? (If you know)